includes/admin/shortcodes/class-shortcode-button.php

```diff
@@ -47,7 +47,10 @@ public function init(){
 
 			add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_assets' ) );
 			add_action( 'admin_enqueue_scripts', array( $this, 'admin_localize_scripts' ), 13 );
-			add_action( 'media_buttons', array( $this, 'shortcode_button' ) );
+
+			if ( true === apply_filters( 'give_enable_shortcode_button', true ) ) {
```

Comment:
@Sidsector9 always add a description to new filter

Reply:
@Sidsector9 I check and found that we already have a filter in code `give_shortcode_button_condition`. Did you notice this?